modules/features/podcasts/src/main/java/au/com/shiftyjelly/pocketcasts/podcasts/view/components/ratings/StarRatingView.kt

```diff
@@ -112,20 +110,15 @@ private fun Content(
 
         Spacer(modifier = Modifier.weight(1f))
 
-        RowOutlinedButton(
+        TextH30(
             text = stringResource(R.string.rate_button),
-            onClick = { onClick(RatingTappedSource.BUTTON) },
-            includePadding = false,
-            fontSize = 16.sp,
-            textPadding = 0.dp,
             fontWeight = FontWeight.W500,
-            border = null,
-            fullWidth = false,
-            colors = ButtonDefaults.outlinedButtonColors(
-                backgroundColor = Color.Transparent,
-                contentColor = MaterialTheme.theme.colors.primaryText01,
-            ),
-            modifier = Modifier.padding(top = 2.dp, bottom = 2.dp),
+            textAlign = TextAlign.Center,
+            fontSize = 16.sp,
+            modifier = Modifier
+                .clickable {
+                    onClick(RatingTappedSource.BUTTON)
+                },
```

Comment:
Could this be better as another component to make the touch area larger and the touch state look nicer.

Before

https://github.com/user-attachments/assets/7917e575-8360-4048-be3d-18171175da14


After

https://github.com/user-attachments/assets/1459432c-2cde-46eb-8ca2-75b0ba6f7694

Reply:
Nice catch! I've improved this here [4799320](https://github.com/Automattic/pocket-casts-android/pull/3513/commits/479932004262c49e2ee1a511b1d0a9ab4942c916)

Reply:
Sorry, but which component are you talking about?

Reply:
The rating stars and rate buttons.